pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -9,29 +9,31 @@ import (
 	"strings"
 	"time"
 
-	"github.com/openshift/machine-api-operator/pkg/util/external"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	"k8s.io/apimachinery/pkg/util/intstr"
-	"k8s.io/klog/v2"
-
-	mapiv1 "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
-	"github.com/openshift/machine-api-operator/pkg/metrics"
-	"github.com/openshift/machine-api-operator/pkg/util/conditions"
```

Comment:
@JoelSpeed when is the best time to do cleanups like this?  As we go, separate commit, or separate PR?

Reply:
just for the record, this is not a cleanup only, I needed to add new import, and there was no clear place where to add it because of the mix of mapi and k8s imports in 2 sections 🙂

Reply:
I'd normally prefer separate commits, especially when it's larger stuff.
As this one is pretty small, and not too difficult to review, I'm happy with it as is.

My general preference in situations like this is to "prefactor", make all the changes to the existing code that are needed as the first few commits, then add the new stuff later, makes the review process as a whole a lot easier in my experience